### DISCUSSION TITLE
Glide and S3

### DISCUSSION THREAD

**Discussion by Christophvh** [statamic/cms#4960] [Q&A] [answered]
Hi, 

I configured my assets to be stored in s3. But I noticed that my glide presets are still stored locally, this kind of makes the whole s3 setup pretty pointless. When we have a lot of presets and a lot of files we are forced to upgrade the server for more storage space. The use of a CDN + s3 also becomes pointless and you lose all of its benefits.  

Is there a way to also store the glide assets in s3?

**Comment by duncanmcclean** [comment, marked as answer]
There's already a feature request open for this - see statamic/ideas#341. Give the issue a 👍  so the Team can sort through the highest voted issues.

**Reply by duncanmcclean** [reply]
Also, seems like there's a draft PR to implement this 👀  #4158
